pkg/network/ovn_kubernetes.go

```diff
@@ -52,6 +52,17 @@ func renderOVNKubernetes(conf *operv1.NetworkSpec, manifestDir string) ([]*uns.U
 		svcpools += fmt.Sprintf("%s", net)
 	}
 	data.Data["OVN_service_cidr"] = svcpools
+	data.Data["OVN_namespace"] = "openshift-ovn-kubernetes"
```

Comment:
Per above, do we really need to template this variable if it's going to be hardcoded?

Reply:
As above it comes up in both configmaps